index.js

```diff
@@ -1,56 +1,87 @@
+// create a class of book.
+const displayContainer = utils.qs('.container');
+let books = localStorage.getItem('books')
+  ? JSON.parse(localStorage.getItem('books'))
+  : [];
+
+const populateBooks = () => {
+  utils.qs('div', displayContainer)?.remove();
+  const div = document.createElement('div');
+  books.forEach((book) => {
+    const wrapper = utils.createElement({});
+
+    wrapper.appendChild(
+      utils.createElement({
+        tagName: 'p',
+        textContent: book.title,
+        class: 'title',
+      }),
+    );
+    wrapper.appendChild(
+      utils.createElement({
+        tagName: 'p',
+        textContent: book.author,
+        title: 'author',
+      }),
+    );
+    wrapper.appendChild(
+      utils.createElement({
+        tagName: 'button',
+        type: 'button',
+        class: 'remove',
+        textContent: 'Remove',
+      }),
+    );
+    wrapper.appendChild(utils.createElement({ tagName: 'hr' }));
+    div.appendChild(wrapper);
+  });
+  displayContainer.appendChild(div);
 };
 
-const getData = () => JSON.parse(localStorage.getItem('books'));
+if (books.length) populateBooks();
 
-const removeBook = (book) => {
-  localStorage.removeItem(book);
+const storeData = () => {
+  localStorage.setItem('books', JSON.stringify(books));
 };
 
-const addBook = (title, author) => {
-  const book = {
-    title,
-    author,
-  };
-  books.push(book);
-  const div = document.createElement('div');
-  books.forEach((book, index) => {
-    div.innerHTML = `
-              <p>${book.title}</p>
-              <p>${book.author}</p>                                          
-              <button type="button" id="${index}" class="remove">Remove</button>   
-              <hr>`;
-    const removebtn = div.querySelector('.remove');
-    removebtn.addEventListener('click', () => {
-      books = books.filter((el) => el.key !== index);
-      div.remove();
-      storeData();
-      removeBook(book);
-    });
-  });
-  storeData();
-  displayContainer.appendChild(div);
+const removeData = (title) => {
+  books.splice([books.findIndex((book) => book.title === title)], 1);
+  books = books.filter((v) => v !== null);
 };
 
-document.querySelector('.add').addEventListener('click', (event) => {
-  event.preventDefault();
-  if (title.value !== '' && author.value !== '') {
-    addBook(title.value, author.value);
-    document.querySelector('form').reset();
-  }
+displayContainer.addEventListener('click', (e) => {
+  const { target } = e;
+
+  if (!target.classList.contains('remove')) return;
+
+  const title = utils.qs('p.title', target.parentElement).textContent;
+
+  removeData(title);
+  storeData();
+  populateBooks();
 });
 
-const display = () => {
-  if (getData()) {
-    getData().forEach((book) => {
-      addBook(book.title, book.author);
-    });
+class Book {
+  constructor(title, author) {
+    this.author = author;
+    this.title = title;
   }
-};
-display();
\ No newline at end of file
+}
```

Comment:
- [ ] Good job so far for creating the class :+1:  but there are some issues you need to address :cry: From the requirement, your class has `methods` to `add `and `remove` books and you should ensure that these two methods inside your class are working fine :+1: 

See an example below :point_down: 

```javascript
class Books {
  constructor() {
    this.books = [];
  }

 // Add book method
  addBook(title, author) {
  }

 // Remove book method
 RemoveBook(bookID) {
  }

 // Get book method
  getBooks() {
    return this.books;
  }
}

```